server/resources/resourcesController.js

```diff
@@ -48,6 +49,8 @@ router
 			const { id: source } = req.user;
 			try {
 				const resource = await service.create({ ...req.body, source });
+				const validated_admin = await usersService.validateAdmin(source);
+				validated_admin && (await service.publish(resource.id, source));
```

Comment:
❌ Don't use this kind of "trick"; it can be helpful in JSX, where you need an _expression_, but in regular JS you can use a proper [_statement_](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else):

```suggestion
        if (validated_admin) {
				  await service.publish(resource.id, source);
				}
```

Reply:
Changes reverted, functionality moved to database